components/ws-manager-api/core.proto

```diff
@@ -32,6 +32,9 @@ service WorkspaceManager {
     // setTimeout changes the default timeout for a running workspace
     rpc SetTimeout(SetTimeoutRequest) returns (SetTimeoutResponse) {}
 
+    // setClosedTimeout changes the default closed timeout for a running workspace
+    rpc SetClosedTimeout(SetClosedTimeoutRequest) returns (SetClosedTimeoutResponse) {}
```

Comment:
This should be part of the SetTimeout call, i.e. extend the `SetTimeoutRequest` appropriately